makePoplogCommander.sh

```diff
@@ -241,6 +412,20 @@ poplog --dev [OPTION]...
     normal mode for programming in Poplog. It is not normally necessary to
     supply this option.
 
+poplog --use-build=(nox|xt|xm) [OPTION]...
```

Comment:
I'm not overly keen on `--use-build`, can it not be `--use-gui-toolkit` or `--use-gui`?

Reply:
Yes. In some ways less elegant (what does `--no-gui --gui=xt` mean?) but definitely much friendlier and easy to understand. The latter is the priority & so we are taking this on :)